var/spack/repos/builtin/packages/hydrogen/package.py

```diff
@@ -107,15 +106,9 @@ def libs(self):
             'libEl', root=self.prefix, shared=shared, recursive=True
         )
 
-    @when('@:0.84' or '@0.99:')
     def cmake_args(self):
         spec = self.spec
 
-        if '@:0.87.7' in spec and '%intel@:17.0.2' in spec:
```

Comment:
is this no longer true?

Reply:
No, it's correct. The problem is `hydrogen` shouldn't actually have releases not covered by it: those are all still `elemental`...